content/rs/references/crdb-cli-reference.md

```diff
@@ -17,7 +17,14 @@ Each instance is responsible for updating the instances residing on other partic
 Write conflicts are resolved using [CRDTs]({{< relref "/rs/administering/active-active.md" >}}).
 
 To programmatically maintain an Active-Active database and its instances, you can use the CRDB-CLI command-line tool.
-With CRDB-CLI you can:
+
+## Using the crdb-cli
+
+To use the CRDB-CLI tool, you must log in to the terminal of an RS host with an account that is a member of the **redislabs** OS group.
```

Comment:
With the customizable user and groups
maybe it's better to say you need to use a member who belongs to that group or if none defined then it's redisLabs